qiskit_optimization/problems/substitute_variables.py

```diff
@@ -154,16 +154,14 @@ def _feasible(sense: ConstraintSense, rhs: float) -> bool:
         """Checks feasibility of the following condition
         0 `sense` rhs
         """
-        # I use the following pylint option because `rhs` should come to right
-        # pylint: disable=misplaced-comparison-constant
         if sense == ConstraintSense.EQ:
-            if 0 == rhs:
+            if rhs == 0:
                 return True
         elif sense == ConstraintSense.LE:
-            if 0 <= rhs:
+            if rhs >= 0:
```

Comment:
I intentionally implemented the conventional code so that the sense (= LE) is consistent with the if clause `0 <= rhs`. If we flip it, they are not consistent. For example, if `sense == ConstraintSense.LE`, we check `rhs >= 0` (>= is GE). I prefer `pylint: disable=misplaced-comparison-constant`.

Reply:
I see `misplaced-comparison-constant` has been moved to an extension of pylint. It does not make sense to activate it only for this case.